features/bundler-cache/devcontainer-feature.json

```
@@ -0,0 +1,17 @@
+{
+  "id": "bundler-cache",
+  "version": "1.0.0",
+  "name": "Bundler cache",
+  "description": "Creates a volume for persisting the installed gems across different containers",
+  "containerEnv": {
+    "BUNDLE_PATH": "/bundle/vendor"
+  },
```

perhaps also add this?

```suggestion
  },
  "installsAfter": [
    "ghcr.io/rails/devcontainer/features/ruby"
  ],
```

I'm not sure if it makes sense though

---

I don't think it's necessary. The ruby feature should not install any gems.